src/main/scala/mesosphere/marathon/core/groupmanager/package.scala

```diff
@@ -0,0 +1,6 @@
+package mesosphere.marathon.core
+
+/**
+  * This package allows to manage all group related actions.
+  */
+package object groupmanager
```

Comment:
I don't see much value - please remove.
Also: please rename the package to `group` instead of `groupmanager`